src/libraries/illink-oob.targets

```diff
@@ -0,0 +1,71 @@
+
+    <Message Text="Trimming $(PackageRID) OOB assemblies with ILLinker..." Importance="high" />
+
+    <PropertyGroup>
+      <LibrariesTrimmedOOBArtifactsPath>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'ILLinkTrimAssembly', '$(BuildSettings)', 'trimmed-oobs'))</LibrariesTrimmedOOBArtifactsPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- add references from the libraries directory -->
+      <_DependencyDirectories Include="$(NetCoreAppCurrentRuntimePath.TrimEnd('\'))" />
+    </ItemGroup>
+
+    <ItemGroup>
+
+      <!-- The following is the list of all the OOBs we will ignore for now -->
+      <_OOBsToIgnore Include="System.CodeDom" />
+      <_OOBsToIgnore Include="System.ComponentModel.Composition" />
+      <_OOBsToIgnore Include="System.ComponentModel.Composition.Registration" />
+      <_OOBsToIgnore Include="System.Composition.AttributedModel" />
+      <_OOBsToIgnore Include="System.Composition.Convention" />
+      <_OOBsToIgnore Include="System.Composition.Hosting" />
+      <_OOBsToIgnore Include="System.Composition.Runtime" />
+      <_OOBsToIgnore Include="System.Composition.TypedParts" />
+      <_OOBsToIgnore Include="System.Configuration.ConfigurationManager" />
+      <_OOBsToIgnore Include="System.Speech" />
+
+      <_OOBsToLink Include="$(NetCoreAppCurrentRuntimePath)*.dll" Exclude="$(NetCoreAppCurrentRuntimePath)*.Generator.dll;$(NetCoreAppCurrentRuntimePath)*.Native.dll" />
+      <_RuntimePackLinkedAssemblies Include="$(MicrosoftNetCoreAppRuntimePackRidLibTfmDir)*.dll" />
+
+      <!-- Move previous items to FileName so that we can subtract them -->
+      <_OOBsToFileName Include="@(_OOBsToLink -> '%(FileName)')">
+        <OriginalIdentity>%(Identity)</OriginalIdentity>
+      </_OOBsToFileName>
+      <_RuntimePackAssembliesToFileName Include="@(_RuntimePackLinkedAssemblies -> '%(FileName)')">
+        <OriginalIdentity>%(Identity)</OriginalIdentity>
+      </_RuntimePackAssembliesToFileName>
+
+      <_FinalOOBsFileName Include="@(_OOBsToFileName)" Exclude="@(_RuntimePackAssembliesToFileName);@(_OOBsToIgnore)" />
+      <_FinalOOBReferences Include="@(_OOBsToFileName)" Exclude="@(_FinalOOBsFileName)" />
+      <_FinalOOBs Include="@(_FinalOOBsFileName -> '%(OriginalIdentity)')" />
+      <_FinalReferences Include="@(_FinalOOBReferences -> '%(OriginalIdentity)')" />
+      <_FinalReferences Include="$(SystemPrivateCoreLibPath)" />
+
+      <OOBRootAssemblies Include="@(_FinalOOBs)">
+        <RootMode>library</RootMode>
+      </OOBRootAssemblies>
+    </ItemGroup>
+
+    <ItemGroup>
+      <!-- Include suppression XML files bin-placed in earlier per-library linker run. -->
+      <_SuppressionsXmls Include="$(ILLinkTrimAssemblyOOBSuppressionsXmlsDir)*.xml" />
+    </ItemGroup>
+
+   <PropertyGroup>
+      <ILLinkArgs Condition="'@(_SuppressionsXmls)' != ''" >$(ILLinkArgs) --link-attributes &quot;@(_SuppressionsXmls->'%(FullPath)', '&quot; --link-attributes &quot;')&quot;</ILLinkArgs>
```

Comment:
I may not have this straight - but since we have 2 targets that are running in the same build, setting common properties in the targets is going to affect the other one, right? 

So for example, if `ILLinkTrimOOBAssemblies` runs first, it is going to set `ILLinkArgs` here.  Then when `ILLinkTrimSharedFramework` runs, it is going to see the stuff the OOB target set in `ILLinkArgs`.

Reply:
great catch, will fix.